plugin/fuzz/source/dextool/plugin/fuzz/backend/fuzzvariant.d

```diff
@@ -407,40 +398,43 @@ body {
 
         auto inner = modules;
         CppModule inner_impl_singleton;
-        if (ns.fullyQualifiedName.toLower in xmlp.getNamespaces()) {
-            //final switch(cast(NamespaceType) ns.kind) with (NamespaceType) {
-            //  case none:
-            inner.hdr = modules.hdr.namespace(ns.resideInNs[0]);
-            inner.impl = modules.impl.namespace(ns.resideInNs[0]);
-            foreach (nss; ns.resideInNs[1 .. $]) {
-                inner.hdr = inner.hdr.namespace(nss);
-                inner.impl = inner.impl.namespace(nss);
-            }
-
-            string fqn_class = ns.fullyQualifiedName; // ~ "::"  ~ class_name;
-
-            foreach (a; ns.classRange) {
-                string class_name = a.name[2 .. $];
-                logger.trace("class_name: " ~ class_name);
-                logger.trace("fqn_class: " ~ fqn_class);
+        //final switch(cast(NamespaceType) ns.kind) with (NamespaceType) {
+        //  case none:
+        inner.hdr = modules.hdr.namespace(ns.resideInNs[0]);
+        inner.impl = modules.impl.namespace(ns.resideInNs[0]);
+        foreach (nss; ns.resideInNs[1 .. $]) {
+            inner.hdr = inner.hdr.namespace(nss);
+            inner.impl = inner.impl.namespace(nss);
+        }
 
-                Namespace nss =  xmlp.getNamespace(ns.fullyQualifiedName.toLower);
+        string fqn_class = ns.fullyQualifiedName; // ~ "::"  ~ class_name;
 
-                foreach (b; a.methodPublicRange) {
-                    if (!(fqn_class in classes) && !(nss.interfaces.ci.empty)) {
-                        classes[fqn_class] = generateClass(inner, class_name,
-                            cast(string[]) ns.resideInNs[0 .. $ - 1],
-                            ns.resideInNs[$ - 1].payload, nss);
-                    }
-                    b.visit!((const CppMethod a) => generateCppMeth(a,
-                        classes[fqn_class], class_name, fqn_class, xmlp),
-                        (const CppMethodOp a) => writeln(""),
-                        (const CppCtor a) => generateCtor(a, inner),
-                        (const CppDtor a) => generateDtor(a, inner));
-                }
+        foreach (a; ns.classRange) {
+            string class_name = a.name[2 .. $];
```

Comment:
The commit you were reviewing was old, so that part of the code has been removed from the latest version, but yes that was ti remove I_